app/models/user.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      firstName: { type: DataTypes.STRING, allowNull: false },
+      lastName: { type: DataTypes.STRING, allowNull: false },
+      email: {
+        type: DataTypes.STRING,
+        unique: true,
+        allowNull: false
+      },
+      password: { type: DataTypes.STRING, allowNull: false }
+    },
+    {}
+  );
+  User.associate = function(models) {
```

Comment:
this goes in classMathods, and you can return the `define` result directly
Like this:
```
module.exports = (sequelize, DataTypes) => {
  return sequelize.define(
    'user',
    {
      firstName: DataTypes.STRING,
      lastName: DataTypes.STRING,
      email: DataTypes.STRING,
      password: DataTypes.STRING
    },
    {
      classMethods: {
        associate(models) {
          // associations can be defined here
        }
      }
    }
  );
};
```
The other thing is that in postgres, tables wont allow upper case so user lower case names instead in the `define`